models/schema/src/plugin.ts

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright © 2024 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import schema, { schemaId } from '@hcengineering/schema-resources/src/plugin'
+import { mergeIds } from '@hcengineering/platform'
+
+export { schemaId } from '@hcengineering/schema'
+export default mergeIds(schemaId, schema, {})
```

Comment:
So, `schema` plugin is effectively empty. Should we drop it and convert `schema-resources` into package, maybe? But keep in mind we still need `model-schema` plugin to register mixin